### DISCUSSION TITLE
Leaflet ground track application

### DISCUSSION THREAD

**Discussion by dylan-baros** [Leaflet/Leaflet#9565] [Q&A]
I am trying to display ground tracks of satellites with leaflet but no matter what I try, the projection looks bad. It's either cutting off the poles or zoomed out to where the map is a square (no wrapping). I want the map to fill the screen and show the entire map. Any advice?

**Comment by IvanSanchez** [comment]
Spend more time with `proj4leaflet`. Otherwise, if you want any advice, it'd be useful to know what you've tried.
